aten/src/ATen/core/ivalue_inl.h

```diff
@@ -82,6 +82,14 @@ inline c10::intrusive_ptr<ivalue::Object> IValue::toObject() const & {
   AT_ASSERT(isObject(), "Expected Object but got ", tagKind());
   return toIntrusivePtr<ivalue::Object>();
 }
+inline c10::intrusive_ptr<ivalue::PythonObject> IValue::toPythonObject() && {
+  TORCH_INTERNAL_ASSERT(isPythonObject(), "Expected PythonObject but got", tagKind());
+  return moveToIntrusivePtr<ivalue::PythonObject>();
+}
+inline c10::intrusive_ptr<ivalue::PythonObject> IValue::toPythonObject() const & {
```

Comment:
This should just directly return python `PyObject*` its a pointer so ivalue.h doesn't need to know its definition.